Disaster-Response-Platform/mobile/DisasterResponsePlatform/app/src/main/java/com/example/disasterresponseplatform/MainActivity.kt

```diff
@@ -91,10 +83,37 @@ class MainActivity : AppCompatActivity() {
         val getEventViewModel: EventViewModel by viewModels()
         eventViewModel = getEventViewModel
 
+        val getEmergencyViewModel: EmergencyViewModel by viewModels()
+        emergencyViewModel = getEmergencyViewModel
+
+        val getResourceViewModel: ResourceViewModel by viewModels()
+        resourceViewModel = getResourceViewModel
+    }
+
+    private fun initializeFragments(){
+        homePageFragment = HomePageFragment(this)
+        networkFragment = NetworkFragment(this)
+        activityFragment = ActivityFragment(needViewModel)
```

Comment:
At MainActivity.kt line 96, it gives an error while compiling.  Error message:
"Too many arguments for public constructor ActivityFragment()"

Reply:
Okay, I missed this because I had files that weren't pushed on my local, thanks. I think I fixed it